.codecov.yml

```diff
@@ -13,5 +13,16 @@ coverage:
 
 ignore:
   - "elasticdl/python/tests"
+  - "elasticdl/python/common/k8s_client.py"
```

Comment:
May we know why should we exclude these source files from unit test coverage stats?